include/flang/lower/PFTBuilder.h

```diff
@@ -317,15 +317,20 @@ struct FunctionLikeUnit : public ProgramUnit {
     return getA<parser::MpSubprogramStmt>();
   }
 
-  std::list<FunctionStatement> funStmts; // begin/end pair
-  EvaluationCollection evals;            // statements
-  std::list<FunctionLikeUnit> funcs;     // internal procedures
+  /// Anonymous programs do not have a begin statement
+  std::optional<FunctionStatement> beginStmt;
+  FunctionStatement endStmt;
+  EvaluationCollection evals;        // statements
+  std::list<FunctionLikeUnit> funcs; // internal procedures
```

Comment:
Interesting change on these two lines.

Reply:
Discussion about this clang-format side-effect moved here: https://github.com/flang-compiler/f18/pull/945#issuecomment-585856610